internal/spdxlicense/license_list.go

```diff
@@ -170,9 +170,9 @@ var licenseIDs = map[string]string{
 	"busl-1":                                   "BUSL-1.1",
 	"busl-1.1":                                 "BUSL-1.1",
 	"busl-1.1.0":                               "BUSL-1.1",
-	"bzip2-1":                                  "bzip2-1.0.5",
-	"bzip2-1.0":                                "bzip2-1.0.5",
-	"bzip2-1.0.5":                              "bzip2-1.0.5",
+	"bzip2-1":                                  "bzip2-1.0.6",
+	"bzip2-1.0":                                "bzip2-1.0.6",
+	"bzip2-1.0.5":                              "bzip2-1.0.6",
```

Comment:
I don't think we should be relabeling these licenses. If something is explicitly `bzip-1.0.5`, we probably shouldn't say it is `1.0.6`

Reply:
Yes we must.

See https://spdx.org/licenses/bzip2-1.0.5.html